app/controllers/api/motorcycles_controller.rb

```diff
@@ -0,0 +1,68 @@
+class Api::MotorcyclesController < ApplicationController
+  before_action :set_user, only: [:create]
+
```

Comment:
- It looks like here you can add a session authentication before entering on this controller (the same for the others), remember, on this capstone project you work with jwt, and for this same reason is a good idea to add the authentications on the controllers. 🐦

Reply:
Thank you for the advice. We however did not consider that since we are a group of 4 and the project requirement did not recommend any strong authentication at the backend. However, we will consider this in our subsequent works. That\nk you once again.